app/models/wiki_content/revision.rb

```diff
@@ -86,4 +86,15 @@ def references_added
     return 0 unless references_count(features) && references_count(features_previous)
     references_count(features) - references_count(features_previous)
   end
+
+  def edit_summary
```

Comment:
I suggest adding a longer comment to this file explaining the switch from edit summaries to diff stats (and the approximate date of that switchover), along with the strategy of using the `summary` field to store diff stats. This is important context that will be helpful for developers who work on anything related to this in the future.